docs/src/main/sphinx/release/release-361.md

```diff
@@ -0,0 +1,88 @@
+# Release 361 (XX Aug 2021)
+
+## General
+
+* Support table redirection for `INSERT`, `DELETE` and `UPDATE` operations. ({issue}`8683`)
+* Improve performance of {func}`sum` and {func}`avg` aggregations involving `decimal` values. ({issue}`8878`)
+* Improve performance for queries using `IN` predicate with moderate to large number of constants. ({issue}`8833`)
+* Fix failures of specific queries accessing ROW columns with fields requiring a name quoting.  ({issue}`8845`)
+
+## Security
+
+* Add support for OAuth2/OIDC opaque access tokens. The property `http-server.authentication.oauth2.audience` has 
+  been removed in favor of using `http-server.authentication.oauth2.client-id` as expected by OIDC. To support 
+  audiences which are not the `client-id` the new property `http-server.authentication.oauth2.additional-audiences` 
+  has been added. Additionally the new property `http-server.authentication.oauth2.issuer` is required; tokens 
+  which are not issued by this URL will be rejected. ({issue}`8641`)
+
+## Cassandra connector
+
+* Add support for pushdown of `smallint`, `tinyint` and `date` types on partition columns. ({issue}`3763`)
+* Fix incorrect results for predicates with inequality on clustering key. ({issue}`401`)
+
+## ClickHouse connector
+
+* Add new configuration property `metadata.cache-maximum-size` to allow limiting the size of the metadata cache. ({issue}`8652`)
+* Fix incorrect results for aggregation functions applied to columns of type VARCHAR and CHAR. ({issue}`7320`)
+
+## Druid connector
+
+* Add new configuration property `metadata.cache-maximum-size` to allow limiting the size of the metadata cache. ({issue}`8652`)
+
+## Hive connector
+
+* Allow access to `$properties` table for Trino and Hive views. ({issue}`8805`)
+* Add support for translating Hive views which use `from_utc_timestamp` function. ({issue}`8502`)
+* Improve performance of updating Glue table statistics for partitioned tables. Also change default 
+  Glue statistics read/write parallelism from 1 to 5. ({issue}`8839`)
+* Improve performance of querying Parquet data. ({issue}`7349`)
+* Fix query failure when inserting data into Hive ACID table which was not explicitly bucketed. ({issue}`8899`)
+
+## Iceberg connector
+
+* Support Iceberg table after partition field dropped. ({issue}`8730`)
+* Allow reading from Iceberg tables which specify `write.object-storage.path` storage property.  ({issue}`8573`)
+* Use randomized location when creating a table, so that later table rename and drop does not interfere with 
+  new tables created with the same name. This can be enabled using `iceberg.unique-table-location` configuration property. ({issue}`6063`)
```

Comment:
```suggestion
* Allow using randomized location when creating a table, so that later table rename and drop does not interfere with 
  new tables created with the same name. This can be enabled using `iceberg.unique-table-location` configuration property. ({issue}`6063`)
```

Reply:
Why "allow"? Is that something the user or admin controls?

Reply:
It felt natural wording given fact that it is not enabled by default.